.github/workflows/publish_images.yaml

```diff
@@ -44,14 +44,25 @@ jobs:
       - name: Build the images
         run: |
           docker build --file pulp_ci_centos/Containerfile --tag pulp/pulp-ci-centos:latest .
+          docker tag pulp/pulp-ci-centos:latest ghcr.io/pulp/pulp-ci-centos:latest
+          docker tag pulp/pulp-ci-centos:latest quay.io/pulp/pulp-ci-centos:latest
           if [ "$VERSION" = "latest" ]; then
             docker build --file pulp_galaxy_ng/Containerfile --tag pulp/pulp-galaxy-ng:latest .
+            docker tag pulp/pulp-galaxy-ng:latest ghcr.io/pulp/pulp-galaxy-ng:latest
+            docker tag pulp/pulp-galaxy-ng:latest quay.io/pulp/all-in-one-galaxy:latest
+
             # Be resilient if this fails
             docker build --file pulp_nightly/Containerfile --tag pulp/pulp:nightly . || true
+            docker tag pulp/pulp:nightly ghcr.io/pulp/pulp:nightly
+            docker tag pulp/pulp:nightly quay.io/pulp/all-in-one-pulp:nightly
```

Comment:
I think, this stops the pipeline if the nightly build failed.

Reply:
nice catch!